share/completions/ssh.fish

```diff
@@ -23,7 +23,7 @@ complete -c ssh -s a --description "Disables forwarding of the authentication ag
 complete -c ssh -s A --description "Enables forwarding of the authentication agent"
 complete -x -c ssh -s b --description "Interface to transmit from" -a "
 (
-	cat /proc/net/arp ^/dev/null| string match -r -v '^IP'|cut -d ' ' -f 1 ^/dev/null
+	tail -n +2 /proc/net/arp | cut -d ' ' -f 1 ^/dev/null
```

Comment:
This removes the stderr redirection, which is useful in case that file doesn't exist.

Also, I'm not sure if this file always has the header in all OSs that have it (just linux, or are there more?).